packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -62,30 +85,69 @@ class CodeSizeBodyState extends State<CodeSizeBody> with AutoDisposeMixin {
       });
     });
 
-    controller.loadTree('v8.json');
+    controller.loadTree('new_v8.json');
   }
 
   @override
   Widget build(BuildContext context) {
     if (root != null) {
-      return Split(
-        axis: Axis.vertical,
+      return Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          _buildTreemap(),
-          _buildTreeTable(),
-        ],
-        initialFractions: const [
-          initialFractionForTreemap,
-          initialFractionForTreeTable,
+          TabBar(
+            isScrollable: true,
+            controller: _tabController,
+            tabs: tabs,
+            onTap: onTabSelected,
+          ),
+          Expanded(
+            child: TabBarView(
+              physics: defaultTabBarViewPhysics,
+              children: _buildTabViews(),
+              controller: _tabController,
+            ),
+          ),
         ],
       );
     } else {
       return const SizedBox();
     }
   }
 
-  Widget _buildTreeTable() {
-    return CodeSizeTable(rootNode: root);
+  void onTabSelected(int index) {
+    final selected = tabs[index].text;
+    // TODO(peterdjlee): Import user file instead of using hard coded data.
+    switch (selected) {
+      case 'Snapshot':
+        controller.loadTree('new_v8.json');
+        return;
+      case 'Diff':
+        controller.loadFakeDiffData();
+        return;
+    }
+  }
+
+  List<Widget> _buildTabViews() {
+    return [
+      _buildTreemapAndTableSplitView(showDiff: false),
+      _buildTreemapAndTableSplitView(showDiff: true),
+    ];
+  }
+
+  Widget _buildTreemapAndTableSplitView({@required bool showDiff}) {
+    return Split(
+      axis: Axis.vertical,
+      children: [
+        _buildTreemap(),
```

Comment:
Add a TODO that says we should try to reuse the same treemap widget and only swap out the tables in Diff mode